scripts/west_commands/sign.py

```diff
@@ -112,6 +112,8 @@ def do_add_parser(self, parser_adder):
                            help='''path to the tool itself, if needed''')
         group.add_argument('-D', '--tool-data', default=None,
                            help='''path to tool data/configuration directory, if needed''')
+        group.add_argument('-X', '--no-xman', action='store_true',
+                           help='''signing image without xman, if needed''')
```

Comment:
maybe just `'''do not add extended manifest'''`

Reply:
Thanks, improve the description.